lib/data/conversation.ts

```
@@ -34,6 +35,40 @@ export type Conversation = typeof conversations.$inferSelect;
 
+const generateEventDescription = (
+  updatedFields: readonly ("status" | "assignedToId" | "assignedToAI")[],
+  previous: typeof conversations.$inferSelect,
+  updated: typeof conversations.$inferSelect,
+): string | null => {
+  const changes: string[] = [];
+
+  for (const field of updatedFields) {
+    switch (field) {
+      case "status":
+        const prevStatus = previous.status || "open";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)